labgraph/examples/graph_w_logging.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
```

Comment:
In the future or now, I can re-structure the example folder. For example, this example can be put in a logging folder.

Reply:
+1, I think that's a great idea.